resources/lang/hu/faults.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+    "faults" => "Hibák",
+    "created_at" => "Időpont",
+    "location" => "Helyszín",
+    "description" => "Leírás",
+    "status" => "Állapot",
+    "submit" => "Küldés",
+    "new_fault" => "Hiba bejelentése",
+    "cancel" => "Mégsem",
+    "confirm_message" => "Biztosan meg akarja változtatni a hiba állapotát a következőre: ",
+    "confirm" => "Megerősítés",
+    "done" => "Kész",
+    "wont_fix" => "Nem megoldandó",
+    "unseen" => "Nem látott",
+    "seen" => "Látott",
+    "reopen" => "Újra",
```

Comment:
újra bejelentett?

Reply:
This text is the one being displayed on the button which reopens an issue, so I don't think that would be an improvement.

Reply:
Added separate texts for reopen _(újranyit)_ and reopened _(újranyitott)_. I've forgotten there were two occurrences.